### PR TITLE
Level 1 - Labrynth

### PR DESCRIPTION
**Pull request by 0xNonCents**
Added a labyrinth of semi-transparent walls to the bottom level of counter splash.
Additionally press 'i' to enter performance.debug mode

Screenshot 2024-06-19 at 11 09 48 PM

Labyrinth design 
Screenshot 2024-06-19 at 11 10 15 PM

### CODE REVIEW

File: src/lib/components/Player.svelte [hunk 1]
```diff
@@ -274,17 +289,15 @@
...
-
   <T.Group position.y={0.9}>
     <T.PerspectiveCamera
       makeDefault
       fov={90}
-      bind:ref={cam}
+      bind:ref={cam}      
       position.x={position[0]}
       position.y={position[1]}
       position.z={position[2]}
```
Comment (0xNonCents) [inline src/lib/components/Player.svelte:299]: Switching this to the store object caused issues so we have position state in the Player component and

File: src/lib/components/Scene.svelte [hunk 2]
```diff
@@ -64,25 +66,23 @@
...
+      <GenericPlaform position={[0, -7, 25]} />
       <!-- Lowest level, easy platforms-->
-      <ObstaclePlatforms />
+      <Level1 />
       <!-- Elevator, a section towards the back that enables vertical movement between the various levels more easily-->
-      <ObstacleObjects seed={3221} center={[0, 40, 70]} containerHeight={60} containerWidth={80} containerLength={40} minSize={5} maxSize={7} numShapes={100} />
+      <ObstacleObjects seed={3221} center={[0, 35, 80]} containerHeight={50} containerWidth={80} containerLength={40} minSize={5} maxSize={7} numShapes={100} />
+      <ObstacleObjects seed={3221} center={[0, 10, 53]} containerHeight={15} containerWidth={20} containerLength={40} minSize={5} maxSize={7} numShapes={12} />
+      <ObstacleObjects seed={3231} center={[-80, 35, 40]} containerHeight={70} containerWidth={60} containerLength={100} minSize={5} maxSize={7} numShapes={100} />
+      <ObstacleObjects seed={3241} center={[80, 35, 40]} containerHeight={70} containerWidth={60} containerLength={100} minSize={5} maxSize={7} numShapes={100} />
```
Comment (0xNonCents) [inline src/lib/components/Scene.svelte:78]: These form the auditorium style obstacles

File: src/lib/components/Player.svelte [hunk 3]
```diff
@@ -116,29 +115,41 @@
...
+      setPlayerPosition([pos.x, pos.y, pos.z])
 
+      console.log($debugMode)
     })
 
     
-
-    const keyMapping: { [x: string]: any; qwerty?: { forward: string; backward: string; left: string; right: string; jump: string }; azerty?: { forward: string; backward: string; left: string; right: string; jump: string }; } = {
+    interface KeyMapping {
+      forward: string;
```
Comment (0xMugen) [inline src/lib/components/Player.svelte:130]: Good clean up

**Review by 0xMugen — COMMENTED** [review]
Debug mode is nice

**Pull request merged**